.github/actions/deploy/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: Deploy to Environment
+description: Build Docker Image, Push it to ECR
```

Comment:
nitpick: this does more than build the image and push it to ECR.

Reply:
updated. i started this before diving into the deploy work.